oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandlerTest.java

```diff
@@ -395,6 +395,7 @@ void shouldUpdateOrchSessionUsingClaimsFromUserInfoResponse()
         assertThat(
                 OrchSessionItem.AccountState.NEW,
                 equalTo(orchSessionCaptor.getAllValues().get(0).getIsNewAccount()));
+        assertTrue(orchSessionCaptor.getAllValues().get(0).getIsAuthenticated());
```

Comment:
Bit of a nitpick but technically this is isn't set from a claim in the userinfo response, so might be better in a new test

Reply:
That is fair, I also think this assertion needs to be added to other tests